Cogs.Console/Program.cs

```diff
@@ -447,11 +447,15 @@ static void Main(string[] args)
                 var namespaceUriPrefix = command.Option("-p|--namespacePrefix",
                                            "Namespace prefix to use for the target Owl namespace",
                                            CommandOptionType.SingleValue);
+                var versionnumber = command.Option("-v|--version",
+                                           "Version number for the target Owl namespace",
+                                           CommandOptionType.SingleValue);
 
                 command.OnExecute(() =>
                 {
                     var location = locationArgument.Value ?? Environment.CurrentDirectory;
                     var target = targetArgument.Value ?? Path.Combine(Directory.GetCurrentDirectory(), "out");
+                    var version = versionnumber.Value() ?? "version1.0";
```

Comment:
Lets default this to the version in the cogsModel.Settings.Version, if it is !string.isnullorwhitespace